apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -368,6 +386,21 @@ const BookingPage = ({
       }
     }
 
+    // Validate that guests are unique
+    let alreadyInvited = false;
```

Comment:
This will remain true for the rest of the guests even if the first guest is the only one repeated. Is this intended?

Reply:
Since this is used under the submit form function, if any guest is repeated then it'll stop the submission of the form and display the form error.